pipelines/vars/forklift_pulpcore.yml

```diff
@@ -0,0 +1,21 @@
+server_box:
+  box: "{{ pipeline_os }}"
+  memory: 4680
+
+forklift_boxes:
+  "{{ {forklift_server_name: server_box} }}"
+
+pulp_install_source: packages
+pulp_pkg_repo: "https://yum.theforeman.org/pulpcore/{{ pipeline_version }}/el{{ ansible_distribution_major_version }}/$basearch/"
+# staging: http://koji.katello.org/releases/yum/pulpcore-{{ pipeline_version }}/el{{ ansible_distribution_major_version }}/$basearch/
```

Comment:
Intentional to leave this commented out here?

Reply:
No, actually, for the pipeline, it should have been the staging repo to begin with ;)